code/modules/jobs/job_types/wasteland.dm

```diff
@@ -339,6 +309,252 @@ Raider
+	gloves = /obj/item/clothing/gloves/f13/handwraps
+	belt = /obj/item/melee/onehanded/knife/hunting
+	r_pocket = /obj/item/flashlight/flare
+	backpack_contents = list(
+		/obj/item/restraints/handcuffs = 2,
+		/obj/item/melee/onehanded/club = 1,
+		/obj/item/reagent_containers/hypospray/medipen/stimpak = 1,
+		/obj/item/storage/bag/money/small/raider = 1,
+		/obj/item/radio/redwater = 1,
+		)
+
+
+/datum/outfit/job/wasteland/f13raider/resident/pre_equip(mob/living/carbon/human/H)
+	. = ..()
+	uniform = pick(
+		/obj/item/clothing/under/f13/merca, \
+		/obj/item/clothing/under/f13/mercc, \
+		/obj/item/clothing/under/f13/cowboyb, \
+		/obj/item/clothing/under/f13/cowboyg, \
+		/obj/item/clothing/under/f13/raider_leather, \
+		/obj/item/clothing/under/f13/raiderrags, \
+		/obj/item/clothing/under/pants/f13/ghoul, \
+		/obj/item/clothing/under/jabroni)
+	if(prob(10))
+		mask = pick(
+			/obj/item/clothing/mask/bandana/red,\
+			/obj/item/clothing/mask/bandana/blue,\
+			/obj/item/clothing/mask/bandana/green,\
+			/obj/item/clothing/mask/bandana/gold,\
+			/obj/item/clothing/mask/bandana/black,\
+			/obj/item/clothing/mask/bandana/skull)
+	shoes = pick(
+			/obj/item/clothing/shoes/jackboots,\
+			/obj/item/clothing/shoes/f13/raidertreads)
+
+	suit_store = pick(
+		/obj/item/gun/ballistic/revolver/detective, \
+		/obj/item/gun/ballistic/automatic/pistol/ninemil,\
+		/obj/item/gun/ballistic/automatic/pistol/m1911, \
+		/obj/item/gun/ballistic/automatic/pistol/type17, \
+		)
+
+
+/datum/outfit/job/wasteland/f13raider/resident/post_equip(mob/living/carbon/human/H, visualsOnly = FALSE)
+	..()
+	if(visualsOnly)
+		return
+	ADD_TRAIT(H, TRAIT_LONGPORKLOVER, src)
+
+	H.social_faction = FACTION_RAIDERS
+	add_verb(H, /mob/living/proc/creategang)
+
+
+/datum/outfit/loadout/raider_sheriff
+	name = "Redwater Peacekeeper"
+	suit = /obj/item/clothing/suit/armor/light/duster/desperado
+	uniform = /obj/item/clothing/under/syndicate/tacticool
+	head = /obj/item/clothing/head/f13/town/big
+	shoes = /obj/item/clothing/shoes/laceup
+	backpack_contents = list(
+		/obj/item/gun/ballistic/rifle/hunting/remington = 1,
+		/obj/item/ammo_box/a308 = 1,
+		/obj/item/ammo_box/a308box = 1,
+		/obj/item/attachments/scope = 1,
+		/obj/item/gun/ballistic/revolver/revolver44=2,
+		/obj/item/storage/belt/holster=1,
+		/obj/item/ammo_box/m44=4,
+		/obj/item/book/granter/trait/gunslinger = 1,
+		/obj/item/card/id/dogtag/sheriff = 1)
+
+/datum/outfit/loadout/raider_mobster
+	name = "Redwater Strongarm"
+	belt = /obj/item/storage/belt/military/assault
+	shoes = /obj/item/clothing/shoes/laceup
+	uniform = /obj/item/clothing/under/f13/densuit
+	suit = /obj/item/clothing/suit/armor/medium/vest
+	gloves =  /obj/item/clothing/gloves/color/white
+	head = /obj/item/clothing/head/fedora
+	neck = /obj/item/storage/belt/holster/ranger44
+	mask =  /obj/item/clothing/mask/bandana/durathread
+	backpack_contents = list(
+		/obj/item/gun/ballistic/automatic/smg/greasegun/mid = 1,
```

Comment:
I totally forgot these were a thing, I havent rebalanced the different crafting tiers of SMGs. Luckily this one's pretty much fine, it doesnt have any vars on it, but most of the others have funky damage multipliers. So, if anyone complains a gun does like 10 times the damage, that's why. id replace it with just a greasegun cus, really, theyre all the same.

Reply:
this is a me-problem btw

Reply:
Yeah I can easily fix that